cucumber-expressions/ruby/spec/cucumber/cucumber_expressions/regular_expression_spec.rb

```diff
@@ -46,6 +46,10 @@ module CucumberExpressions
           "I can cancel the 1st slide upload")
         ).to eq(["I", "can", 1, "slide"])
       end
+      
+      it "ignores escaped 2.x parenthesis" do
+        expect( match(/Across the line\(s\)/, 'Across the line\(s\)') ).to be_nil
```

Comment:
I struggled it for a bit, but thank you for the guidance! I definitely will.

I'm not fully certain about the regex part, perhaps another negative lookahead instead?